skeleton/add_my-great-app_to_apps.json

```diff
@@ -0,0 +1,12 @@
+// Create an entry in apps.json as follows:
```

Comment:
I wonder whether we could make this a README.md that describes what `skeleton` is and where to put the files?

Reply:
Good point will add one